spec/system/submission_details_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe "A11y", :js do
+  describe "Logged-in as a Challenge Manager" do
+    let(:user) { create_user(role: "challenge_manager") }
+    let(:challenge) { create_challenge(user: user, title: "Boston Tea Party Cleanup") }
+    let(:submission) { create(:submission, manager: user, challenge: challenge) }
+
+
+    before { system_login_user(user) }
+
+    it "submission details page is accessible" do
+      visit submission_path(submission)
+      expect(user.role).to eq("challenge_manager")
+      expect(page).to have_content(submission.id)
```

Comment:
I'd like to use more specific rules to ensure we are testing the correct content on the page, validating unique content. I added the `have_css` selector for request specs (e20849c), and I'd like to take advantage of it by checking the css as well as page content match. in this case:
```suggestion
      expect(page).to have_css('h1', text: "Submission ID #{submission.id}")
```
the other tests look good to me for page interaction specification 👍